lib/ResolverFactory.js

```diff
@@ -199,7 +202,7 @@ function createOptions(options) {
 		}
 	}
 
-	return {
+	return normalizeOptionsForWindows({
```

Comment:
I am afraid it will be slow on big builds...

Reply:
Okay, I'll take a look how it integrates with other libraries, maybe I'll find better place to normalize it